zedd-app/src/ClarityState.ts

```diff
@@ -33,6 +40,15 @@ export class ClarityState {
    */
   public resourceName: string | undefined
 
+  @observable
+  public error: string = ""
```

Comment:
If you initialize the variable with a default value, the type is not necessary: public error = ''